index.html

```diff
@@ -0,0 +1,932 @@
+          </p>
+        </a>
+      </div>
+
+      <div class="latest-deal-box">
+        <div class="latest-deal-img bg-3"></div>
+        <a class="a-tags-latest-news underline change-color-red" href="https://thenextweb.com/apps/2020/07/30/blender-microsoft-free-development-fund/">
+          <p class="latest-deal-title"> Enpass Password Manager<br>
+            <span class="latest-deal-price-scrape">$59</span>
+            <span class="latest-deal-price"> $24.99</span>
+          </p>
+        </a>
+      </div>
+
+      <div class="latest-deal-box">      
+        <div class="latest-deal-img bg-4"></div>
+        <a class="a-tags-latest-news underline change-color-red" href="https://thenextweb.com/apps/2020/07/30/blender-microsoft-free-development-fund/">
+          <p class="latest-deal-title"> KeepSolid VPN Unlimited<br>
+            <span class="latest-deal-price-scrape"> $199</span><span class="latest-deal-price"> $39</span>
+          </p>
+        </a>
+      </div>
+
+      <div class="latest-deal-box">
+        <div class="latest-deal-img bg-5"></div>
+        <a class="a-tags-latest-news underline change-color-red" href="https://thenextweb.com/apps/2020/07/30/blender-microsoft-free-development-fund/">
+        <p class="latest-deal-title">LongTailPro: One Time <br>
+          <span class="latest-deal-price-scrape">$1500</span><span class="latest-deal-price"> $49.99</span>
+        </p>
+        </a>
+      </div>
+
+      <div class="latest-deal-box">
+        <div class="latest-deal-img bg-6"></div>
+        <a class="a-tags-latest-news underline change-color-red" href="https://thenextweb.com/apps/2020/07/30/blender-microsoft-free-development-fund/">
+          <p class="latest-deal-title">StackSkills Unlimited: Lifetime Access <br>
+            <span class="latest-deal-price-scrape">$1495</span><span class="latest-deal-price"> $56</span>
+          </p>
+        </a>
+      </div>
+
+      <div class="latest-deal-box">
+        <div class="latest-deal-img bg-7"></div>        
+        <a class="a-tags-latest-news underline change-color-red" href="https://thenextweb.com/apps/2020/07/30/blender-microsoft-free-development-fund/">
+          <p class="latest-deal-title">VPN by VeePN: Subscription <br>
+            <span class="latest-deal-price-scrape">$659</span><span class="latest-deal-price"> $29.99</span>
+          </p>
+        </a>
+      </div>
+
+      <div class="latest-deal-box">
+        <div class="latest-deal-img bg-8"></div>
+        <a class="a-tags-latest-news underline change-color-red" href="https://thenextweb.com/apps/2020/07/30/blender-microsoft-free-development-fund/">
+        <p class="latest-deal-title">Ivacy VPN: Lifetime Subscription <br>
+          <span class="latest-deal-price-scrape">$1194</span><span class="latest-deal-price"> $39.99</span>
+        </p>
+        </a>
+      </div>
+    </div>
+
+  </section>
+
+  <footer class="footer">
```

Comment:
There is a blank space below the footer. It's more prominent in tablet and mobile view.

![image](https://user-images.githubusercontent.com/57726348/89258964-e9fdca80-d646-11ea-8c5c-328a3a18fd88.png)